homeassistant/components/tradfri/__init__.py

```diff
@@ -32,18 +32,15 @@
     COORDINATOR,
     COORDINATOR_LIST,
     DOMAIN,
+    FACTORY,
     KEY_API,
+    LISTENERS,
     PLATFORMS,
     SIGNAL_GW,
     TIMEOUT_API,
 )
 from .coordinator import TradfriDeviceDataUpdateCoordinator
 
-_LOGGER = logging.getLogger(__name__)
-
-FACTORY = "tradfri_factory"
-LISTENERS = "tradfri_listeners"
-
 CONFIG_SCHEMA = cv.removed(DOMAIN, raise_if_present=False)
```

Comment:
While a constant, I've seen elsewhere in the codebase that this should stay here and not be moved to `const.py`?

Reply:
The const.py module is for common constants used in more than one module. Local constants are better kept in the module where they are used.

Yes, we should keep this one here.

Reply:
Ok! Maybe I should move some constants in the other direction then? 🤔

Reply:
Yeah, it's good if we're consistent.

Reply:
Will do!

Reply:
Should I also annotate all constants with `Final` while I'm at it?

Reply:
Only if it's needed.

Reply:
It works as is now so then I’ll leave it. Might change opinion when we bump pytradfri later on.